.changeset/khaki-pianos-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Adds new utilities for building apps for the dev toolbar more easily
```

Comment:
Since it's a minor, should it show a few code snippets?

Reply:
Yep, for sure.